meinberlin/migrations/0001_initial.py

```diff
@@ -20,8 +20,28 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='HomePage',
             fields=[
-                ('page_ptr', models.OneToOneField(auto_created=True, on_delete=django.db.models.deletion.CASCADE, parent_link=True, primary_key=True, serialize=False, to='wagtailcore.Page')),
-                ('description', wagtail.wagtailcore.fields.RichTextField(blank=True)),
+                ('page_ptr', models.OneToOneField(auto_created=True,
+                    on_delete=django.db.models.deletion.CASCADE,
+                    parent_link=True, primary_key=True, serialize=False,
+                    to='wagtailcore.Page')),
+                ('header', models.CharField(max_length=255, blank=True)),
+                ('description', models.CharField(max_length=255, blank=True)),
+                ('info1_title', models.CharField(max_length=255, blank=True)),
+                ('info1_text', models.CharField(max_length=255, blank=True)),
+                ('info2_title', models.CharField(max_length=255, blank=True)),
+                ('info2_text', models.CharField(max_length=255, blank=True)),
+                ('info3_title', models.CharField(max_length=255, blank=True)),
+                ('info3_text', models.CharField(max_length=255, blank=True)),
+                ('netiquette_linktext', models.CharField(max_length=255, blank=True)),
+                ('privacy_linktext', models.CharField(max_length=255, blank=True)),
+                ('processes_linktext', models.CharField(max_length=255, blank=True)),
+                ('current_processes_linktext', models.CharField(max_length=255, blank=True)),
```

Comment:
this element and some of the following have wring names. I'll rebase and squash.